src/Visualisation-App.tsx

```diff
@@ -1,32 +1,72 @@
-import Viewer from "./viewer";
+import React from "react";
+import Viewer from "./Viewer";
+import ViewerOptions from "./ViewerOptions";
+import "./Visualisation-App.css";
 import { useState } from "react";
-import { searchResultsHits } from "./searchResultsHits.interface";
-import Searchbar from "./searchbar";
-import SearchResults from "./searchResults";
+import Searchbar from "./Searchbar";
+import SearchResults from "./SearchResults";
+import { PluginContext } from "molstar/lib/mol-plugin/context";
+import { SearchResultsHits } from "./SearchResults";
 
-function VisualisationApp() {
+const VisualisationApp: React.FunctionComponent = () => {
     const [url, setUrl] = useState("");
+    // const url = "https://alphafold.ebi.ac.uk/files/AF-P04350-F1-model_v1.cif";
+    const plugin =
+        React.useRef<PluginContext | null>() as React.MutableRefObject<PluginContext | null>;
+
     const [searchResults, setSearchResults] = useState<
-        searchResultsHits[] | null
+        SearchResultsHits[] | null
     >(null);
 
     return (
         <>
             <h1>AlphaFold Protein Viewer</h1>
-            <Searchbar setSearchResults={setSearchResults}></Searchbar>
-            <br />
-            <div>
-                {searchResults && (
-                    <SearchResults
-                        searchResults={searchResults}
-                        url={url}
-                        setUrl={setUrl}
-                    ></SearchResults>
+            <div className="container">
+                <div className="searchArea">
+                    <Searchbar setSearchResults={setSearchResults}></Searchbar>
+                </div>
+                <div className="searchResults">
+                    {searchResults && (
```

Comment:
Might be worth hiding the searchResults div completely if searchResults is false-y, then you don't need the additional `<></>`.